ModelValidator.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name = 'ModelValidator'
+  s.version = '0.1.0'
+  s.summary = 'Framework for model validations'
+  s.description = <<-DESC
+    ModelValidator is a framework for validating models with custom validations
+     errors and returning all errors at once.
+  DESC
+  s.homepage = 'https://github.com/railwaymen/ModelValidator'
+  s.license = { :type => 'MIT', :file => "LICENSE" }
+  s.author = { 'Bartłomiej Świerad' => 'bartlomiej.swierad@railwaymen.org' }
+  s.source = { :git => 'https://github.com/railwaymen/ModelValidator.git', :tag => s.version.to_s }
+  s.source_files = 'Sources/**/*'
+  s.swift_version = '5.2'
+
```

Comment:
Any min iOS deployment target?

Reply:
No, SwiftPM limits only OSX version but works for every version of other systems.

Reply:
https://developer.apple.com/documentation/swift_packages/supportedplatform